src/main/java/chess/repository/ChessRoomRepository.java

```diff
@@ -22,6 +22,22 @@ public ChessRoomRepository(JdbcTemplate jdbcTemplate) {
         this.jdbcTemplate = jdbcTemplate;
     }
 
+    @Override
+    public List<Room> findAll() {
+        String sq = "SELECT * FROM room r JOIN board b on r.board_id=b.id";
```

Comment:
```suggestion
        String sq = "SELECT * FROM room r JOIN board b on r.board_id = b.id";
```

띄어쓰거나 붙여쓰거나 전체적으로 통일해주는게 좋겠네요

Reply:
양식 전부 `A = B` 형식으로 통일했습니다!